react/OrderForm.tsx

```diff
@@ -77,11 +80,17 @@ const getLocalOrderForm = (): OrderForm | null => {
 
 export const OrderFormProvider: FC = ({ children }) => {
   const { logSplunk } = useSplunk()
+  const runtime = useRuntime
+  const page = runtime().page
+
   const { loading, data, error } = useQuery<{
     orderForm: OrderForm
   }>(OrderFormQuery, {
     ssr: false,
     fetchPolicy: 'no-cache',
+    variables: {
+      refreshOutdatedData: page.includes(CHECKOUT)
```

Comment:
have you checked that if you navigate from, say, a product page to the cart page this query will be remade? or do we need to add an `useEffect` and manually call `refetch`?

Reply:
so far in all the tests done it is being called